digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -806,6 +815,37 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
     }
   }
 
+  put("/linearassets/verified") {
+    val user = userProvider.getCurrentUser()
+    val ids = (parsedBody \ "ids").extract[Set[Long]]
+    val typeId = (parsedBody \ "typeId").extractOrElse[Int](halt(BadRequest("Missing mandatory 'typeId' parameter")))
+    val usedService = getLinearAssetService(typeId)
+    val linkIds = usedService.getPersistedAssetsByIds(typeId, ids).map(_.linkId)
+    roadLinkService.fetchVVHRoadlinks(linkIds.toSet)
+      .map(_.municipalityCode)
+      .foreach(validateUserMunicipalityAccess(user))
+
+    usedService.updateVerifiedInfo(ids, user.username, typeId)
+  }
+
+  get("/linearassets/unverified"){
+      val user = userProvider.getCurrentUser()
+      val includedMunicipalities = if (user.isOperator()) Set() else user.configuration.authorizedMunicipalities
+
+      val typeId = params.getOrElse("typeId", halt(BadRequest("Missing mandatory 'typeId' parameter"))).toInt
+    val usedService = getLinearAssetService(typeId)
+    usedService.getUnverifiedLinearAssets(typeId, includedMunicipalities.toSet)
+  }
+
+  get("/linearassets/midpoint"){

```

Comment:
This code is the same as `getLinearAssetById` on `Digiroad2Api`. This entrypoint does the same as `get("/linearAsset/unchecked/:id")`
  
Probably we should change the name of the unchecked endpoint.